src/spin_sdk/mqtt.py

```diff
@@ -0,0 +1,18 @@
+"""Module for utilizing Spin Outbound MQTT"""
+
+from enum import Enum
+from spin_sdk.wit.imports.mqtt import Connection, Qos as Qos 
+
+def open(address: str, username: str, password: str, keep_alive_interval_in_secs: int) -> Connection:
+    """
+    Open a connection to the Mqtt instance at `address`.
+    
+    A `spin_sdk.wit.types.Err(spin_sdk.wit.imports.redis.ErrorInvalidAddress)` will be raised if the connection string is invalid.
```

Comment:
Why is the `redis` module showing up here?

Reply:
Copy paste without paying attention. Will fix.